open-metadata-implementation/user-interfaces/presentation-server/cra-server/.env_sample

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0 
+# Copyright Contributors to the ODPi Egeria project. 
+
+# This file is a sample .env file that can be used by developers.
+# Copy this file to a file called .env and modify the values as required
+# Each line configures a UI server (tenant) with the appropriate downstream server
+#    
+# In this example the first line configures a ui server called aaa with a remote server name of "cocoView1" and URL of "https://localhost:9443"
+EGERIA_PRESENTATIONSERVER_SERVER_aaa={"remoteServerName":"cocoView1","remoteURL":"https://localhost:9443"}
```

Comment:
When we build an image of this in a container, it should be easy to get this env var -- presumably that is all you are doing with .env. -- initializing the environment?

Reply:
Yes, I believe this is correct